### PR TITLE
License renewal

### PR DESCRIPTION
**Pull request by kengio**

### CODE REVIEW

File: skel/daemons/license.go [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+func getLicenseFromVault(vaultService *vault.Service) (string, error) {
+	app := "finex"
+	scope := "secret"
+
+	// Load secret
+	vaultService.LoadSecrets(app, scope)
+
+	// Get secret
+	license, err := vaultService.GetSecret(app, "license", scope)
```
Comment (vshatravenko) [inline skel/daemons/license.go:177]: Let's use finex_license_key as the field name here as in https://github.com/openware/openfinex/blob/02142e24f2af8555087f669bf79ccab90a996802/pkg/config/config.go#L16

File: skel/daemons/license.go [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+}
+
+func saveLicenseToVault(vaultService *vault.Service, license string) error {
+	app := "finex"
+	scope := "secret"
+
+	// Load secret
+	vaultService.LoadSecrets(app, scope)
+
+	err := vaultService.SetSecret(app, "license", license, scope)
```
Comment (vshatravenko) [inline skel/daemons/license.go:192]: Same here

**Review by vshatravenko — CHANGES_REQUESTED** [review]

File: skel/daemons/license.go (hunk 1 above)
Comment (kengio) [reply skel/daemons/license.go]: Renamed.

File: skel/daemons/license.go (hunk 2 above)
Comment (kengio) [reply skel/daemons/license.go]: Renamed.

**Pull request merged**